variables.tf

```diff
@@ -62,7 +62,7 @@ variable "project_labels" {
 
 variable "project_prefix" {
   description = "Name prefix to use for projects created."
-  default     = "cft"
+  default     = "prj"
```

Comment:
We should keep the default here as is. We have added the `prj` default in example-foundations https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/variables.tf#L61